package.json

```diff
@@ -45,6 +46,7 @@
     "mean-logger": "~0.0.1",
     "mocha-lcov-reporter": "^1.3.0",
     "mongoose": "^5.1.2",
+    "nodemailer": "^4.6.5",
```

Comment:
Let's uninstall the `nodemailer` package

Reply:
Okay fortune, I will do it now. Thank you.